docs/sources/observability-as-code/provision-resources/provisioned-dashboards.md

```diff
@@ -16,9 +16,8 @@ weight: 300
 # Work with provisioned dashboards
 
 {{< admonition type="caution" >}}
-Git Sync and File path provisioning an [experimental feature](https://grafana.com/docs/release-life-cycle/) introduced in Grafana v12 for open source and Enterprise editions. Engineering and on-call support is not available. Documentation is either limited or not provided outside of code comments. No SLA is provided. Enable the `provisioning` and `kubernetesDashboards` feature toggles in Grafana. These features aren't available publicly in Grafana Cloud yet. Only the cloud-hosted version of GitHub (GitHub.com) is supported at this time. GitHub Enterprise is not yet compatible.
 
-Sign up for Grafana Cloud Git Sync early access using [this form](https://forms.gle/WKkR3EVMcbqsNnkD9).
+TBC
```

Comment:
Also need to update this.

I would have ignored these if the PR was in draft but because it's ready to review, I want to make sure they aren't missed before merge.